{
  "uid" : "6a825afdb4ce9f2c",
  "name" : "User can register property in approved state - Tennessee as Entity with mailing address in approved state",
  "fullName" : "tests.test_add_property.TestAddProperty#test_user_can_register_property_in_approved_state_as_entity",
  "historyId" : "ff04afb93bc7306f0b327e4d48c64184",
  "time" : {
    "start" : 1615907181035,
    "stop" : 1615907193631,
    "duration" : 12596
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n            failed while waiting 10 seconds\n            to assert Text\n            for first_by('css selector', '.login__title')\n\n            reason: ConditionMismatchException: condition did not match\n            \texpected: Good evening, Autotest John\n            \t  actual: Good afternoon, Autotest John\n            screenshot: file:///home/runner/.selene/screenshots/1615907173187/screen_1615907173188.png",
  "statusTrace" : "venv/lib/python3.8/site-packages/selene/wait.py:35: in wait_for\n    return condition.fn(entity)\nvenv/lib/python3.8/site-packages/selene/conditions.py:164: in fn\n    return self.match(element.get_actual_webelement())\nvenv/lib/python3.8/site-packages/selene/conditions.py:255: in match\n    raise ConditionMismatchException(expected=self.expected_text, actual=actual_text)\nE   selene.exceptions.ConditionMismatchException: condition did not match\nE               \texpected: Good evening, Autotest John\nE               \t  actual: Good afternoon, Autotest John\n\nDuring handling of the above exception, another exception occurred:\nvenv/lib/python3.8/site-packages/selene/elements.py:227: in _wait_with_screenshot\n    return wait_for(entity, condition, timeout, polling)\nvenv/lib/python3.8/site-packages/selene/wait.py:50: in wait_for\n    raise TimeoutException('''\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 10 seconds\nE               to assert Text\nE               for first_by('css selector', '.login__title')\nE   \nE               reason: ConditionMismatchException: condition did not match\nE               \texpected: Good evening, Autotest John\nE               \t  actual: Good afternoon, Autotest John\n\nDuring handling of the above exception, another exception occurred:\ntests/test_add_property.py:51: in test_user_can_register_property_in_approved_state_as_entity\n    DashboardPage().check_user_greeting_text_is_correct(first_name)\npages/dashboard_page.py:40: in check_user_greeting_text_is_correct\n    greeting.should(have.text(f\"Good evening, {first_name}\"))\nvenv/lib/python3.8/site-packages/selene/elements.py:355: in should\n    _wait_with_screenshot(self._webdriver, self, condition, timeout)\nvenv/lib/python3.8/site-packages/selene/elements.py:232: in _wait_with_screenshot\n    raise TimeoutException(msg, e.screen, e.stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 10 seconds\nE               to assert Text\nE               for first_by('css selector', '.login__title')\nE   \nE               reason: ConditionMismatchException: condition did not match\nE               \texpected: Good evening, Autotest John\nE               \t  actual: Good afternoon, Autotest John\nE               screenshot: file:///home/runner/.selene/screenshots/1615907173187/screen_1615907173188.png",
  "flaky" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1615907173715,
      "stop" : 1615907173767,
      "duration" : 52
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "open_base_url",
    "time" : {
      "start" : 1615907173767,
      "stop" : 1615907181033,
      "duration" : 7266
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open url",
      "time" : {
        "start" : 1615907178595,
        "stop" : 1615907181033,
        "duration" : 2438
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "url",
        "value" : "'https://portal.fyve-cloud.com/'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "inject_config",
    "time" : {
      "start" : 1615907173767,
      "stop" : 1615907173767,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n            failed while waiting 10 seconds\n            to assert Text\n            for first_by('css selector', '.login__title')\n\n            reason: ConditionMismatchException: condition did not match\n            \texpected: Good evening, Autotest John\n            \t  actual: Good afternoon, Autotest John\n            screenshot: file:///home/runner/.selene/screenshots/1615907173187/screen_1615907173188.png",
    "statusTrace" : "venv/lib/python3.8/site-packages/selene/wait.py:35: in wait_for\n    return condition.fn(entity)\nvenv/lib/python3.8/site-packages/selene/conditions.py:164: in fn\n    return self.match(element.get_actual_webelement())\nvenv/lib/python3.8/site-packages/selene/conditions.py:255: in match\n    raise ConditionMismatchException(expected=self.expected_text, actual=actual_text)\nE   selene.exceptions.ConditionMismatchException: condition did not match\nE               \texpected: Good evening, Autotest John\nE               \t  actual: Good afternoon, Autotest John\n\nDuring handling of the above exception, another exception occurred:\nvenv/lib/python3.8/site-packages/selene/elements.py:227: in _wait_with_screenshot\n    return wait_for(entity, condition, timeout, polling)\nvenv/lib/python3.8/site-packages/selene/wait.py:50: in wait_for\n    raise TimeoutException('''\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 10 seconds\nE               to assert Text\nE               for first_by('css selector', '.login__title')\nE   \nE               reason: ConditionMismatchException: condition did not match\nE               \texpected: Good evening, Autotest John\nE               \t  actual: Good afternoon, Autotest John\n\nDuring handling of the above exception, another exception occurred:\ntests/test_add_property.py:51: in test_user_can_register_property_in_approved_state_as_entity\n    DashboardPage().check_user_greeting_text_is_correct(first_name)\npages/dashboard_page.py:40: in check_user_greeting_text_is_correct\n    greeting.should(have.text(f\"Good evening, {first_name}\"))\nvenv/lib/python3.8/site-packages/selene/elements.py:355: in should\n    _wait_with_screenshot(self._webdriver, self, condition, timeout)\nvenv/lib/python3.8/site-packages/selene/elements.py:232: in _wait_with_screenshot\n    raise TimeoutException(msg, e.screen, e.stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message: \nE               failed while waiting 10 seconds\nE               to assert Text\nE               for first_by('css selector', '.login__title')\nE   \nE               reason: ConditionMismatchException: condition did not match\nE               \texpected: Good evening, Autotest John\nE               \t  actual: Good afternoon, Autotest John\nE               screenshot: file:///home/runner/.selene/screenshots/1615907173187/screen_1615907173188.png",
    "steps" : [ {
      "name" : "Submit phone number",
      "time" : {
        "start" : 1615907181110,
        "stop" : 1615907182706,
        "duration" : 1596
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "phone",
        "value" : "'4759900000'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Submit otp code",
      "time" : {
        "start" : 1615907182706,
        "stop" : 1615907183183,
        "duration" : 477
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "code",
        "value" : "'1234'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    }, {
      "name" : "Check user greeting text is correct",
      "time" : {
        "start" : 1615907183183,
        "stop" : 1615907193630,
        "duration" : 10447
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n            failed while waiting 10 seconds\n            to assert Text\n            for first_by('css selector', '.login__title')\n\n            reason: ConditionMismatchException: condition did not match\n            \texpected: Good evening, Autotest John\n            \t  actual: Good afternoon, Autotest John\n            screenshot: file:///home/runner/.selene/screenshots/1615907173187/screen_1615907173188.png\n\n",
      "statusTrace" : "  File \"/home/runner/work/fyve_web/fyve_web/venv/lib/python3.8/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/home/runner/work/fyve_web/fyve_web/pages/dashboard_page.py\", line 40, in check_user_greeting_text_is_correct\n    greeting.should(have.text(f\"Good evening, {first_name}\"))\n  File \"/home/runner/work/fyve_web/fyve_web/venv/lib/python3.8/site-packages/selene/elements.py\", line 355, in should\n    _wait_with_screenshot(self._webdriver, self, condition, timeout)\n  File \"/home/runner/work/fyve_web/fyve_web/venv/lib/python3.8/site-packages/selene/elements.py\", line 232, in _wait_with_screenshot\n    raise TimeoutException(msg, e.screen, e.stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "first_name",
        "value" : "'Autotest John'"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ {
      "uid" : "4d2e992f782d5b80",
      "name" : "Screenshot",
      "source" : "4d2e992f782d5b80.png",
      "type" : "image/png",
      "size" : 75412
    } ],
    "parameters" : [ ],
    "stepsCount" : 3,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "open_base_url::0",
    "time" : {
      "start" : 1615907194173,
      "stop" : 1615907194562,
      "duration" : 389
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "26e60ab1b73f853a",
      "name" : "Screenshot",
      "source" : "26e60ab1b73f853a.png",
      "type" : "image/png",
      "size" : 75412
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Add Property"
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('open_base_url')"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_add_property"
  }, {
    "name" : "subSuite",
    "value" : "TestAddProperty"
  }, {
    "name" : "host",
    "value" : "fv-az252-624"
  }, {
    "name" : "thread",
    "value" : "4100-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_add_property"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ "@pytest.mark.usefixtures('open_base_url')" ]
  },
  "source" : "6a825afdb4ce9f2c.json",
  "parameterValues" : [ ]
}